static/index.html

```diff
@@ -147,42 +147,48 @@
                 </li>
                 <li>
                     <label>KSP-Version<br /><input type="text" id="ksp_version" size="40" /></label> examples: <code>==1.0</code> or <code>&lt;=1.1&gt;=1.0.3</code>
+                    <br />
+                    <small>While not technically mandatory, not providing this field is considered bad practice.</small>
```

Comment:
Worth noting that if/when https://github.com/KSP-CKAN/CKAN/pull/1742 is merged this will be technically mandatory.